src/Tooltip/TooltipTouch.tsx

```diff
@@ -22,6 +32,10 @@ export default class TooltipTouch extends React.PureComponent {
     isOpen: false
   };
 
+  private ref: HTMLDivElement
+
+  timeout: NodeJS.Timer
```

Comment:
this should be `: number`... please, make sure you're using `window.setTimeout` instead of `setTimeout` (by default, typescript is assuming we want to use the `setTimeout` from Node)

Reply:
almost: tsc correctly resolves window.sentiment, but vscode resolves the node one. Using window.setTimeout makes everyone happy